recipes/buttons/callbar_button_with_popover/callbar_button_with_popover.stories.js

```diff
@@ -16,8 +16,10 @@ const iconsList = getIconNames();
 // Default Prop Values
 export const argsData = {
   buttonWidthSize: 'xl',
+  openPopover: false,
   onArrowClick: action('arrowClick'),
   onClick: action('click'),
+  onModalIsOpened: action('opened'),
```

Comment:
Replace it with 
`onOpened: action('opened'),`

Reply:
It is not necessary to have two `// Action Event Handlers`, you can unified both and add this code:

```
 opened: {
    table: {
      disable: false,
      type: {
        summary: 'event',
      },
    },
  },
```

```
onOpened: {
    table: {
      disable: true,
    },
  },
```